Package.swift

```diff
@@ -309,6 +309,7 @@ let package = Package(
         .product(name: "SwiftRefactor", package: "swift-syntax"),
         .product(name: "SwiftSyntax", package: "swift-syntax"),
         .product(name: "SwiftToolsSupport-auto", package: "swift-tools-support-core"),
+        .product(name: "SwiftPM-auto", package: "swift-package-manager"),
```

Comment:
I think you need to add the corresponding dependency to the CMake build as well.

Reply:
I must be missing something. The CMake targets for other things that use SwiftPM don't list a specific dependency on anything in SwiftPM.

Reply:
If it builds, whatever. I don’t know how this CMake build works.